Demos/DatAgora_PartDieu/ud-viz-context/DatAgora_PartDieu/assets/icons/3dtilesDebug.svg

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```

Comment:
These assets file are copies of `https://github.com/VCityTeam/UD-Viz-demo/tree/master/DemoFull/assets`...